src/meta/server_state.cpp

```diff
@@ -2864,5 +2883,79 @@ void server_state::clear_app_envs(const app_env_rpc &env_rpc)
                    new_envs.c_str());
         });
 }
+
+namespace {
+
+bool validate_target_max_replica_count_internal(int32_t max_replica_count,
+                                                int32_t alive_node_count,
+                                                std::string *hint_message)
+{
+    dassert_f(FLAGS_min_allowed_replica_count <= FLAGS_max_allowed_replica_count,
```

Comment:
just return error, but not dassert, make meta crush is not good

Reply:
Actually this check has been done during validator, so I think we can just remove it ?
```C++
DSN_DEFINE_validator(min_allowed_replica_count, [](int32_t allowed_replica_count) -> bool {
    return allowed_replica_count > 0 && allowed_replica_count <= FLAGS_max_allowed_replica_count;
});
```